editor/src/clj/editor/scene_picking.clj

```diff
@@ -41,3 +44,21 @@
                       (coll/some #(and (not= % (:node-id node))
                                        (contains? node-ids %)))))]
     (e/remove child? nodes)))
+
+(defn contains-resource?
+  [project acc-fn node-id resource]
```

Comment:
I think we should put the `acc-fn` as the last argument, since it may be rather long if supplied directly to `contains-resource?`.